osd/aws/InstallFailed_TooManyBuckets.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Installation blocked, action required",
+    "description": "cluster's installation blocked on AWS TooManyBuckets error. Please raise the S3 bucket quota or remove some buckets from the appropriate region of your AWS account for the installation to succeed. AWS account limits are described in this document -  https://docs.openshift.com/container-platform/4.6/installing/installing_aws/installing-aws-account.html#installation-aws-limits_installing-aws-account",
```

Comment:
What about replacing the URL with https://docs.openshift.com/container-platform/latest/installing/installing_aws/installing-aws-account.html#installation-aws-limits_installing-aws-account ?

Changing 4.6 to latest. Each version might have some change/difference from the previous one.

Reply:
sounds good... done